pkg/logutil/log.go

```diff
@@ -83,6 +84,30 @@ func SetLogLevel(level string) error {
 	return nil
 }
 
+// InitLoggerForCDC initializes logger for CDC, including grpc, sarma etc.
+func InitLoggerForCDC(cfg *Config) error {
```

Comment:
Can we make the `grpc/sarama` related logical as options of `InitLogger` instead of defining a new `InitLoggerForCDC` interface because it is not extensible?
In this way, we will add more and more functions like this if we want to initialize more environments.

Prefer:
```go
func InitLogger(cfg *Config, opts ...LoggerOpt) error {}
```

Reply:
Fixed.